src/lib.rs

```diff
@@ -827,21 +852,33 @@ impl<T: Counter> Histogram<T> {
         };
 
         if !recorded_without_resize {
-            if !self.auto_resize {
+            if clamp {
+                value = if value > self.highest_trackable_value {
+                    self.highest_trackable_value
+                } else {
+                    // must be smaller than the lowest_discernible_value, since self.mut_at(value)
+                    // failed, and it's not too large (per above).
+                    self.lowest_discernible_value
+                };
+
+                // unwrap must succeed since low and high are always representable
+                let c = self.mut_at(value).unwrap();
```

Comment:
I'm not sure about the convention on these things but is it perhaps preferable to have this be an `expect()` with the info in the comments as the argument?